src/Tools/RepoUtil/RepoUtil.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>RepoUtil</RootNamespace>
     <AssemblyName>RepoUtil</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFramework>net46</TargetFramework>
     <RuntimeIdentifiers>win7</RuntimeIdentifiers>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <TargetFrameworkProfile />
```

Comment:
There are more properties that can go away either because they are defaulted or not used anymore. We can also convert to file globs and get rid of the individual file includes - (it is a big time saver for merges\switching branches). That should be a separate PR though. I'll volunteer to do that (will be good testing for my [tool](https://github.com/srivatsn/ProjectSimplifier)) as long as you don't object to globs.